core/device_impl.cpp

```diff
@@ -164,7 +164,7 @@ void DeviceImpl::process_statustext(const mavlink_message_t &message)
     mavlink_statustext_t statustext;
     mavlink_msg_statustext_decode(&message, &statustext);
 
-    std::string debug_str = "mavlink ";
+    std::string debug_str = "MAVLink ";
```

Comment:
May seem finicky, but can I have a colon - `MAVLink: `. Just makes it clear that this is an "origin/source".

Reply:
Done.